content/concepts/introduction/software-ecosystem.md

```diff
@@ -33,6 +36,8 @@ Core servers expose [HTTP / WebSocket APIs](http-websocket-apis.html) for users
 
 Libraries simplify some of the common work of accessing the XRP Ledger, usually through the HTTP / WebSocket APIs. They convert the data into forms that are more familiar and convenient for various programming languages, and include implementations of common operations. Some client libraries are officially maintained by the XRP Ledger Foundation and some are maintained by other entities in the community.
```

Comment:
```suggestion
Libraries simplify some of the common work of accessing the XRP Ledger, usually through the HTTP / WebSocket APIs. They convert the data into forms that are more familiar and convenient for various programming languages, and include implementations of common operations. 
```

Reply:
Afaik, the client libraries listed on xrpl.org are all under the XRPLF org in GitHub.